tap_github/user_streams.py

```diff
@@ -181,24 +181,25 @@ def query(self) -> str:
               repositoriesContributedTo (first: 100 includeUserRepositories: true orderBy: {field: STARGAZERS, direction: DESC}) {
                 nodes {
                   node_id: id
-                  nameWithOwner
-                  openGraphImageUrl
-                  stargazerCount
+                  name_with_owner: nameWithOwner
+                  open_graph_image_url: openGraphImageUrl
```

Comment:
I think this has some other name in rest

Reply:
I'll leave it as is for now.